modules/extensions-spring-support/src/test/resources/reconnection-config.xml

```diff
@@ -32,7 +32,11 @@
     </flow>
 
     <flow name="stickyPagedOperation">
-        <reconnection:sticky-paged-operation failOn="#[payload]" withErrorType="#[vars.errorType]" config-ref="reconnection_Config"/>
+        <reconnection:sticky-paged-operation failOn="#[payload]" config-ref="reconnection_Config"/>
```

Comment:
you can't remove the `withErrorType` parameter. Otherwise you're tweaking the test to be green rather than fixing the issue.